.cicd/generate-pipeline.sh

```diff
@@ -551,7 +552,7 @@ EOF
 fi
 # pipeline tail
 cat <<EOF
-  - wait:
+  - wait: ~
```

Comment:
What does the tilde (`~`) do here?

Reply:
That is the YAML null symbol. In this case, it enforces that you cannot add information at the end.

https://buildkite.com/docs/pipelines/wait-step

Reply:
> The explicit null `~` character used in the above examples isn't required, but is recommended as a best practice. It ensures that nothing else is accidentally added to the `wait` before the `continue_on_failure` attribute.

Hmmm, interesting. I had never seen this before. Thank you.